models/db.js

```diff
@@ -0,0 +1,17 @@
+const mysql = require('mysql');
+const util = require('util');
+
+const pool = mysql.createPool({
+    host: process.env.DB_HOST || 'fakehost',
```

Comment:
Should we remove the || 'fakehost' and the other three || 'fake'? I looked at the mysql documentation and it looks like its not standard? I am unsure

Reply:
We probably want to use `.env` to supply these at all times anyways, no preference either way, I can remove

Reply:
Yea lets remove it. Sorry for originally saying to add it.